norse/torch/functional/spikes_to_times_decoder.py

```diff
@@ -0,0 +1,98 @@
+import torch
+
+
+class SpikesToTimesDecoder(torch.nn.Module):
+    """
+    Module wrapper for spike to spike-time (or spike-index) decoder.
+
+    Decodes input spikes with shape [time_steps x batch_size x output_size] into spike indices
+    tensor with shape [spike_count x batch_size x output_size] in ascending order, e.g. for
+    spike_count = 1 returns only times of first spikes along each batch- and output-dimension.
+    If stated, converts indices along time axis into times with given dt value.
+    """
+
+    def __init__(
+        self,
+        spike_count: int = 1,
+        convert_indices_to_times: bool = True,
+        dt: float = 1e-3,
+    ):
+        """
+        Initialize decoder.
+        :param spike_count: number of elements (ascending) to return for each output channel.
```

Comment:
We use a different convention of specifying parameters in docs strings, could you please follow the convention based on what is done in other files.